adminer/static/editing.js

```diff
@@ -283,6 +283,30 @@ function editingRemoveRow(button, name) {
 	return true;
 }
 
+/** Move table row for field
+* @param HTMLInputElement
+* @param bool direction
+* @return boolean
+*/
+function editingMoveRow(button, dir){
+	var row = parentTag(button, 'tr');
+	var table = row.parentNode;
+	var tags = table.getElementsByTagName('tr');
+	for (var i=0; i < tags.length; i++) {
```

Comment:
Because text nodes are dom elements too (Including the whitespace between TR tags)

If we could ensure the TRs and the TBODY are always rendered without whitespace between them we could use nextSibling no problem, otherwise it might seem like the buttons are broken when it only reorders no-op whitespace